### ISSUE TITLE
[Feature Request] Mod to remove lyrics

### ISSUE THREAD

**Issue by subheeroo** [pannal/Sub-Zero.bundle#642]
The mods in Sub-Zero are great, however there's one mod that I would really appreciate that's missing. I don't prefer HI subtitles and only use regular ones. For some reason many subtitles still include lyrics from songs that are playing which is very distracting during an intense scene. 

Would it be possible to include the removal of lyrics in the Remove HI feature, or possibly as its own mod? 

Lyrics lines usually begin or end with either ♪ or ♫.

**Comment by pannal** [comment]
Yes I can add that. It has the potential to possibly delete some false-positive lines, as subtitle creators not always end their musical lines properly, but it's certainly doable.

I didn't include this in the remove_HI filter, because, honestly, I like to see the lyrics as well :D

**Comment by subheeroo** [comment]
> Yes I can add that. It has the potential to possibly delete some false-positive lines, as subtitle creators not always end their musical lines properly, but it's certainly doable.
> 
> I didn't include this in the remove_HI filter, because, honestly, I like to see the lyrics as well :D

Thanks for your fast reply! I'm looking forward to the update! 
Even if they will break some lines I still prefer it without lyrics. 

I can see the appeal sometimes but most of the time the lyrics just annoy me, hehe.

**Comment by pannal** [comment]
@subheeroo want to try the test branch with the feature enabled by default?

https://github.com/pannal/Sub-Zero.bundle/tree/submod_opt

**Comment by subheeroo** [comment]
> @subheeroo want to try the test branch with the feature enabled by default?
> 
> https://github.com/pannal/Sub-Zero.bundle/tree/submod_opt

I'm running this on my Nvidia Shield and don't have access to the plugin files so I'm afraid I can't test it out. Sorry! I hope you can get it verified otherwise.

**Comment by pannal** [comment]
@subheeroo you can try [this](https://github.com/pannal/Sub-Zero.bundle/issues/606).
